gui/qt/main_window.py

```diff
@@ -1252,6 +1251,19 @@ def get_custom_fee_text(self, fee_rate = None):
             if fee_rate is None: fee_rate = self.config.custom_fee_rate() / 1000.0
             return str(round(fee_rate*100)/100) + " sats/B"
 
+    @staticmethod
+    def output_for_opreturn_stringdata(op_return):
+        if op_return:
+            if not isinstance(op_return, str):
+                raise OPReturnError('OP_RETURN parameter needs to be of type str!')
+            elif len(op_return) > 220:
```

Comment:
Are you sure this is correct?  You are checking the string literal for a length of 220.  My understanding is that this does not always translate into 220 bytes when encoded via utf-8.  If you are using an esoteric character that takes more than 1 byte, this check could pass but the payload will be too large.

Reply:
Hey I actually fixed this in a subsequent commit.  Not sure how you ended up on an older commit my man.  Do a git pull...